scripts/asos/iem_scraper_example.py

```diff
@@ -134,3 +176,18 @@ def main():
 if __name__ == "__main__":
     download_alldata()
     # main()
+
+    
+'''
+Example use case of using the construct_urls and pull_metar functions. 
+
+def main():
+    start = (2021,5,1)
+    finish = (2021,11,30)
```

Comment:
You wanna try hooking these up to a `sys.argv` parsing?

Reply:
I was going to use sys.argv, but found that using argparse was potentially a better usability for this kind of input.